move-vm-backend/src/lib.rs

```diff
@@ -116,4 +116,28 @@ where
 
         self.warehouse.apply_changes(changeset)
     }
+
+    pub fn execute_script(
+        &self,
+        script_bytecode: &[u8],
+        type_args: Vec<TypeTag>,
+        args: Vec<&[u8]>,
+        gas: &mut impl GasMeter,
+    ) -> Result<(), Error> {
+        let mut sess = self.vm.new_session(&self.warehouse);
+
+        let res = sess
```

Comment:
`res` seems unused here
```suggestion
        sess
```

Reply:
Done.